packages/modules/bigcommerce/src/factories/get-transformed-page-data.ts

```diff
@@ -1,9 +1,16 @@
-import { NormalPage, RawHtmlPage } from '@aligent/bigcommerce-operations';
+import { SeoDetailsFragment } from '@aligent/bigcommerce-operations';
 import { CmsPage } from '@aligent/bigcommerce-resolvers';
 
 const CND_MASK = /%%GLOBAL_CdnStorePath%%/g;
 
-export const getTransformedPageData = (data: NormalPage | RawHtmlPage, cdnUrl: string): CmsPage => {
+type PageData = {
+    path: string;
+    htmlBody: string;
+    name: string;
+    seo: SeoDetailsFragment;
```

Comment:
Should we have the SeoDetailsFragments as a manual type too?

Reply:
@jackmacaa the fragment type is generated from our actual request to bigcommerce, so it won't break things when unrelated types change on the schema.

Ideally we would use fragments for the page types as well but that's not how the GraphQL requests were architected and I wanted to avoid runtime changes (which modifications to the GraphQL requests would be!)

Reply:
Ah ye that makes sense.